src/api/CustomAction/types.ts

```diff
@@ -109,6 +109,10 @@ export interface CustomAction extends PersistedItem, SharingOptions {
   description: string;
   parsedTemplate: ParsedCustomAction;
   right: CustomActionRight;
+  owner?: {
```

Comment:
@maxime-allex why optional? you should add a comment in which case it's optional (or make it mandatory)

Reply:
I thought we wanted to return `owner` only if the custom action was shared (GraphQuery interface misled me).